GameLib/GameField.cpp

```diff
@@ -5,16 +5,19 @@
 #include <cstddef>
 #include <list>
 #include <set>
+#include <stdexcept>
+#include <utility>
 
 using Combinatorics::Edge;
 using Combinatorics::Graph;
 
+// static std::string const delimiter_player = "X";
 static std::string const delimiter_x_open = "   ";
-static std::string const delimiter_x_closed = " | ";
-static std::string const delimiter_x_barrier_check = "S";
+static std::string const delimiter_x_closed = " |  ";
```

Comment:
```suggestion
static std::string const delimiter_x_closed = " | ";
```

der eine space verschiebt sonnst das spielfeld in der ausgabe. (glaub ich, teste das mal.)